srcDLL/DllMain.cpp

```diff
@@ -143,12 +141,13 @@ void LocateVolFiles(const std::string& relativeDirectory)
 			const auto extension = ToLower(filePath.extension().string());
 
 			if (extension == ".vol") {
-				volList->AddVolFile((fs::path(relativeDirectory) / filePath.filename()).string());
+				auto relativePath = fs::relative(absoluteDirectory, GetExeDirectory()) / filePath.filename();
+				volList->AddVolFile(relativePath.string());
```

Comment:
We could generalize this method by using some sort of callback for what to do with the found path.

Reply:
See if the function I placed in the FileSystem library satisfies your intention. It sounds like you want to go a step further and act on results instead of return a container. I would accomplish maybe by passing a function as an argument or something... Not sure I want to go that route right now due to technical difficulty accomplishing on my part.